polars/polars-core/src/chunked_array/temporal/datetime.rs

```diff
@@ -106,7 +106,7 @@ impl DatetimeChunked {
             let chunks = self
                 .downcast_iter()
                 .map(|arr| {
-                    replace_timezone(arr, self.time_unit().to_arrow(), to, from, use_earliest)
+                    replace_timezone(arr, self.time_unit().to_arrow(), from, to, use_earliest)
```

Comment:
So this was a bug before, right?

Reply:
it's not a bug, I'm just refactoring a bit. the names were swapped inside `replace_timezone`

it worked, it was just very confusing to read

there's no change in behaviour here

Reply:
There's no update to `replace_timezone` though? Or am I missing something?

Reply:
you're right, the update is to `convert_to_naive_local` (which `replace_timezone` gets to)